### PR TITLE
doctl-account: add page

### PR DESCRIPTION
**Pull request by 258204**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #7475

### CODE REVIEW

File: pages/common/doctl-account.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# doctl account
+
+> Retrieve information about digital ocean accounts.
```
Comment (CleanMachine1) [inline pages/common/doctl-account.md:3]:
```suggestion
> Retrieve information about Digital Ocean accounts.
```

File: pages/common/doctl-account.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/account/>.
+
+- Show available commands:
+
+`doctl account --help`
+
```
Comment (CleanMachine1) [inline pages/common/doctl-account.md:9]: Could you please move this command to be the last?

File: pages/common/doctl-account.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- Show account info:
```
Comment (CleanMachine1) [inline pages/common/doctl-account.md:10]:
```suggestion
- Display account info:
```

File: pages/common/doctl-account.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/common/doctl-account.md:9]: And make - Show available commands: 

to be 

- Display help:

File: pages/common/doctl-account.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`doctl account get`
+
+- Show the hourly API limit, progress towards it, and when the limit resets:
```
Comment (CleanMachine1) [inline pages/common/doctl-account.md:14]:
```suggestion
- Show the hourly API limit, progress towards it, and when the rate limit resets:
```

What do you think about this suggestion, I feel the sentence reads a little strange without it.

**Review by CleanMachine1 — COMMENTED** [review]
Thanks, here are a few suggestions.

**Review by blueskyson — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
Looks good to me

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**